src/axios-har-tracker.ts

```diff
@@ -96,17 +168,42 @@ async function axiosTracker(call) {
             _blocked_queueing: -1
           }
         };
-        const enteriesContent = Object.assign({}, newEntry);
-        generatedHar.log.entries.push(enteriesContent);
+        this.entriesContent = Object.assign({}, this.newEntry);
+        this.generatedHar.log.entries.push(this.entriesContent);
         return resp;
       }
-    },
-    error => {
-      return Promise.reject(error);
+    );
+  }
+
+  private errorHandler(error){
+    return Promise.reject(error);
+  }
+
+  private resetNewEntry () {
```

Comment:
this.newEntry body has to defined two times, once in the constructor and second in resetNewEntry. Maybe it would be better to have one method `generateNewEntry` which will return new entry object and just call this method wherever is needed.